test/choosetests.jl

```diff
@@ -80,10 +80,52 @@ function choosetests(choices = [])
         prepend!(tests, linalgtests)
     end
 
+    sparsetests = ["sparse/sparse", "sparse/sparsevector"]
+    if Base.USE_GPL_LIBS
+        append!(sparsetests, ["sparse/umfpack", "sparse/cholmod", "sparse/spqr"])
+    end
+    if "sparse" in skip_tests
+        filter!(x -> (x != "sparse" && !(x in sparsetests)), tests)
+    elseif "sparse" in tests
+        # specifically selected case
+        filter!(x -> x != "sparse", tests)
+        prepend!(tests, sparsetests)
```

Comment:
don't think these sets take long enough that they're important to put at the end